tools-of-quality.md

```diff
@@ -0,0 +1,112 @@
+# 🛠 Tools of Quality
+
+Our working environment should help us do high quality work because of it, rather than in spite of it.
+
+This document attempts to be an overview of what we need and want in such an environment, without prescribing concrete products. Preferences will be noted.
+
+## 👀 Transparency
+
+To ensure a wholesome and productive environment, we _must and will_ be fully transparent with the team.
+
+The customer is a part of the team. Therefore, if you "can't" say something to the customer, it does not exist for the team. We will not be forced to keep secrets from our own team members.
+
+In the same spirit, we will not do any double bookkeeping of issues, statuses, time etc. If this is "needed", we will have no part in it and the work will fall to whoever feels that this is "needed".
+
+## 💬 Language
+
+We strive to be as clear and concise as possible.
+
+For the sake of clarity we forgo the use of abbreviations unless these are widely understood (we will, for example, use `URL` instead of `Uniform Resource Locator`).
+
+We will write code, documentation, issues etc. in US English, as this is the language of the code we write, and makes it possible to include those not proficient in Norwegian as members of a team.
+
+If the customer demands that some of these things be done in Norwegian, an explicit choice must be made to satisfy that demand.
+
+Certain circumstances exist where mixing of languages is preferred, such as when a domain is only easily expressed in a given language.
+
+Regardless of language, we will strive to write and speak in a gramatically correct manner in order to avoid misunderstandings.
```

Comment:
Is this perhaps a bit superflous? 
I think we can grammar good and can learn to do other stuff good too?

Reply:
You'd be surprised at how many people don't care at all. "Everyone knows what I mean" etc.

Reply:
See, for example, the whole "skjema" -> "schema" issue (found in several projects e ven though it doesn't make sense at all) 😱.

Reply:
:D